cmd/server.go

```diff
@@ -54,8 +56,7 @@ var serverCmd = &cobra.Command{
 		ctx, cancel := context.WithCancel(context.Background())
 		GracefulExit(cancel)
 
-		var running sync.WaitGroup
-		running.Add(2)
+		var group errgroup.Group
```

Comment:
Let's change this to use [`errgroup.WithContext`](https://godoc.org/golang.org/x/sync/errgroup#WithContext) so we can do proper cleanup. Assigning it over the existing `ctx` variable is fine.